packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    v-if="banner"
```

Comment:
do not use the v-if on the root component, it is the responsibility of the parent component to decide when to show this or not show this, the v-if should be used there